tests/functional/db/cram/load-query-export.t

```diff
@@ -0,0 +1,36 @@
+Setup
+
+  $ export AUGUR="$TESTDIR/../../../../bin/augur"
+
+Load a metadata file.
+
+  $ cat >in.tsv <<~~
+  > strain	date
+  > SEQ1	2000-01-01
+  > SEQ2	2000-02-01
+  > SEQ3	2000-03-01
+  > SEQ4	2000-04-01
+  > ~~
+
+  $ ${AUGUR} db load \
+  >  --metadata in.tsv \
+  >  --to-sqlite3 data.sqlite3
+
+Query the metadata and create a new table in the database.
+
+  $ sqlite3 data.sqlite3 "
+  >   CREATE TABLE filtered_metadata AS
+  >   SELECT strain FROM metadata
+  >   WHERE strain in ('SEQ1', 'SEQ3')
+  > "
```

Comment:
This is pretty slick! It hints at the power of loading these data into a database for all kinds of uses...